tools/scripts/ffi.toml

```diff
@@ -67,24 +74,38 @@ cd ffi/cpp/docs;
 exec --fail-on-error make html
 '''
 
-[tasks.gen-c-diplomat]
+[tasks.diplomat-gen-c]
 description = "Generate C headers for the FFI with Diplomat"
 category = "ICU4X Development"
 script_runner = "@duckscript"
 script = '''
 exit_on_error true
 cd ffi/capi
+# Duckscript doesn't support globs in rm, so we delete the dir
+rm -r include/
+mkdir include
 exec --fail-on-error diplomat-tool c ./include
 '''
 
-[tasks.gen-cpp-diplomat]
+[tasks.diplomat-gen-cpp]
 description = "Generate C++ headers for the FFI with Diplomat"
 category = "ICU4X Development"
 script_runner = "@duckscript"
 script = '''
 exit_on_error true
 cd ffi/capi
-exec --fail-on-error diplomat-tool cpp ../cpp/include
+
+
+# Duckscript doesn't support globs in rm, so we do this instead
+include = glob_array ../cpp/include/**/*.hpp
+for path in ${include}
+    rm ${path}
+end
+docs = glob_array ../cpp/docs/source/*.rst
+for path in ${docs}
+    rm ${path}
+end
```

Comment:
Suggestion (optional): Why not `rm` the whole directory like you did for C and JS?

Reply:
There are files we care about in here, like gitignores and conf.py